frontend/src/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+import { DayHeaders } from './DayHeaders'
+import { MonthlyContainer } from './MonthlyContainer'
+import { eventDays, processedDates } from './../../store/stubs'
+
+const CalenderContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: min-content;
+  margin-right: auto;
+  margin-left: auto;
+`
+
+const YearContainer = styled.div`
+  width: calc(47.14px * 7);
+  font-weight: 600;
+  color: #58b994;
+  text-align: center;
+  padding-top: 10px;
+`
+
+const MonthContainer = styled.div`
+  padding-bottom: 5px;
+`
+
+const MonthsHeaderContainer = styled.div`
+  width: 93px;
+  color: #58b994;
+  margin-top: 10px;
+  margin-left: 10px;
+  padding-bottom: 20px;
+  font-weight: 600;
+`
+
+const DatesGridContainer = styled.div`
+  display: grid;
+  grid-template-columns: 47.14px 47.14px 47.14px 47.14px 47.14px 47.14px 47.14px;
+  grid-template-rows: 40px 40px 40px 40px 40px 40px;
+`
+
+// this component takes in an array of events or an array of dates that has events
+export const Calendar = () => {
+  // conert unix dates into component friendly format; 302 for 2nd of March, 1105 for 5th Nov
```

Comment:
Will this be used throughout the feature?
Consider moving this type of "reusable" methods out into a common.ts files.

Reply:
Fixed in 100644